internal/backend/db/dbgorm/builds_test.go

```diff
@@ -52,6 +52,42 @@ func TestDeleteBuild(t *testing.T) {
 	f.assertBuildDeleted(t, b.BuildID)
 }
 
+func TestDeleteProjectDeleteBuild(t *testing.T) {
+	f, p := preBuildTest(t)
+	b := f.newBuild(p)
+	f.saveBuildsAndAssert(t, b)
+
+	// test deleteProject
+	assert.NoError(t, f.gormdb.deleteProject(f.ctx, p.ProjectID))
+	f.assertBuildDeleted(t, b.BuildID)
+}
+
+func TestDeleteBuildDeleteDeployment(t *testing.T) {
```

Comment:
That's kinda scary - how do we make sure we don't delete an active deployment?

Reply:
This is a test file, and we should never run tests in deployments.